Vizzuality/gfw-components
# Feature/header-component

## Improvements

1. added type button for buttons
2. Fixed bug caused when mixing buttons and links, buttons need 1px margin to align correctly 

Before: 
<img width="200" alt="Screenshot 2020-10-22 at 14 45 31" src="https://user-images.githubusercontent.com/971129/96873488-49fd5380-1475-11eb-80fc-2c63ffdfeeaf.png">

After:
<img width="200" alt="Screenshot 2020-10-22 at 14 45 36" src="https://user-images.githubusercontent.com/971129/96873511-541f5200-1475-11eb-9023-2358ead034b5.png">

3. Add green hover + active border 
4. Make menu fixed width on non-fullscreen devices. 
5. Make search in submenu sticky if overflow scroll is active. 
6. Removed DS_Store from repo + added to gitignore
7. As submenu is not a "dropdown" menu, added functionality to automatically close it, if outside of boundary. 
8. Center links in dropdown menus as reflected in design. 
9. Remove legal and added "contact" to more gfw section

## Test instructions

1. Test that menu works correctly on mobile/desktop 
2. Check that fullscreen (map vs rest) works as expected
3. Check different languages and make sure menu is working correctly 
4. Added "contact" to more gfw section, check that clicking that one opens the correct action
